src/utils/csv_writer.py

```diff
@@ -5,7 +5,7 @@
 def export_json_file_to_csv(file_name, OUTPUT_DIR):
     os.makedirs(f'{OUTPUT_DIR}/json', exist_ok=True)
     df = pd.read_json(f'{OUTPUT_DIR}/json/{file_name}.json').transpose()
-    df = df.applymap(convert_array_to_string)
+    df = df.map(convert_array_to_string)
```

Comment:
funnily enough mine only works with `applymap`. What pandas version are you on?

Reply:
I just re-installed it to double check, the warning is ```
F:\Repositories\deadbot\src\utils\csv_writer.py:8: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  df = df.applymap(convert_array_to_string)```
Can you double check your version is up to date? I would presume that if applymap is deprecated, that's the older one.

Reply:
It's probably cause the version in pyproject.toml is 2.0.1. Just use that version for now, we can upgrade it in another pr

Reply:
> It's probably cause the version in pyproject.toml is 2.0.1. Just use that version for now, we can upgrade it in another pr

How do I replace with an older version? My dependency installation process is still screwed up, needa use python3 -m for everything, so wanna make sure I get it right :/

Reply:
pip is still pip, whether it has python3 before or not. You should be able to find the syntax to pip install a specific version

Reply:
> pip is still pip, whether it has python3 before or not. You should be able to find the syntax to pip install a specific version

Fair enough, but the syntax isn't my problem. This PR can wait until after the pandas version is updated in the pyproject.tml.

Reply:
Well you did ask how to install it. Either way, this PR should have no changes related to pandas. Just remove the change and let the other issue handle it, meaning this won't be blocked.

Also needs updating with develop as there should be no output-data anymore